### PR TITLE
added Device Trait "Cook"

### PR DESCRIPTION
**Pull request by GreatSUN**
added Device Trait Cook to support emulating "intelligent coffee machines" as well.

### CODE REVIEW

File: io-package.json [hunk 1]
```diff
@@ -1,8 +1,20 @@
 {
   "common": {
     "name": "iot",
-    "version": "1.11.4",
+    "version": "1.11.5",
```
Comment (Apollon77) [inline io-package.json:4]: Please do not increase version and do not add news entries (especially not empty ones!!) ... this is done by release script based on redme changelog entries

File: src/src/Tabs/Options.js [hunk 2]
```diff
@@ -126,10 +131,7 @@ class Options extends Component {
                 <CardContent>{Utils.renderTextWithA(I18n.t('amazon link'))}</CardContent>
             </CardActionArea>
             <CardActions style={{textAlign: 'center'}} >
-                <Button variant="outlined" size="small" color="primary" style={{textAlign: 'center'}} onClick={() => {
-                    const win = window.open('https://alexa.amazon.de/spa/index.html#skills/dp/B07L66BFF9/reviews', '_blank');
-                    win.focus();
-                }}>{I18n.t('Review')}</Button>
+                <Button variant="outlined" size="small" color="primary" style={{textAlign: 'center'}} onClick={() => { openAmazonReviewUrl() }}>{I18n.t('Review')}</Button>
```
Comment (Apollon77) [inline src/src/Tabs/Options.js:134]: what does this change do ? Code is basically the same??!

File: io-package.json (hunk 1 above)
Comment (Apollon77) [reply io-package.json:4]: SO please add a new README changelog entry

**Review by Apollon77 — CHANGES_REQUESTED** [review]

**Comment by GreatSUN** [comment]
Need to get rid of some stuff and redo changes properly ;-)

Sorry!

**Pull request closed without merging**